modules/ROOT/pages/_partials/commons/common-query-n1ql-mobile.adoc

```diff
@@ -140,10 +140,10 @@ This behavior is inline with that of Couchbase Server {sqlpp_cbm} -- see example
 | SELECT * FROM _default
 | _default
 
-|SELECT * FROM db
-|db
+|SELECT * FROM datasource
```

Comment:
I couldn't find where we describe what the datasource could be. In general, the datasource code be the following:

* < database-name > : default collection
* _ (underscore)       : default collection
* < scope-name >.< collection-name > : a collection in a scope
* < collection-name > : a collection in the default scope

I have also provided some example in one of the comments in https://issues.couchbase.com/browse/CBL-4054.